shared/actions/chat/shared.js

```diff
@@ -181,11 +181,11 @@ function _isTimestampableMessage (message: Constants.Message): boolean {
   return (!!message && !!message.timestamp && !['Timestamp', 'Deleted', 'Unhandled', 'InvisibleError', 'Edit'].includes(message.type))
 }
 
-function _previousTimestampableMessage (messages: Array<Constants.Message>, prevIndex: number): ?Constants.Message {
-  return findLast(messages, message => _isTimestampableMessage(message) ? message : null, prevIndex)
+function _previousTimestampableMessage (messages: List<Constants.Message>, prevIndex: number): ?Constants.Message {
```

Comment:
maybe we should do the opposite (especially if we're just going to call toArray on it). And instead if you call this thing with a List you have to do toArray() first